# ContourList.msg

geometry_msgs/Point32[] contour
Contour[] contours